src/generate.js

```diff
@@ -97,4 +141,37 @@ for (const repo of repos) {
 
   fs.writeFileSync(generatedConfigPath, generatedJson);
   console.log(`Generated tasks written to ${generatedConfigPath}`);
+
+  if (notifications.Markdown.length > 0) {
+    const generatedMarkdown = path.join(generatedFolder, `${repo}.md`);
```

Comment:
I recommend having the generated markdown files included in git within this repo as well. It's made reviewing how changes affect generated output easier to accomplish in PRs here before opening PRs in the target repos.

We should probably restructure the `generated` folder to use:
- `generated/{repo}/.github/fabricbot.json`
- `generated/{repo}/docs/area-owners.md`